### PR TITLE
Update to parametric values in prognostic convection for GFSv17

### PR DESCRIPTION
**Pull request by lisa-bengtsson**
These PR’s in ufs-weather-model, fv3atm and ccpp/physics does the following:
Creates a set of _gfsv17 RT’s to reflect tests that are beyond prototype 8 targeted for GFSv17
Activates prognostic closure by setting namelist progsigma = true in _gfsv17 and pointing to new field_table in the RT’s
Addresses additional parametric tuning in progclosure_calc.F90
Includes TKE contribution from cu for progsigma area fraction in samfdeep and samfshal cu schemes. 
Adds a new gfsv17 field_table and diag_table 

Issue: https://github.com/ufs-community/ufs-weather-model/issues/1477
Dependencies: https://github.com/ufs-community/ufs-weather-model/pull/1480 , https://github.com/NOAA-EMC/fv3atm/pull/598 

Here is a PPT describing the impact of this tuning on slide 9: https://docs.google.com/presentation/d/1m8GNn03tkwuKxuQxIt4kRcih5Ta3JDtQHy79VMBW20Y/edit#slide=id.g17f3245d7e2_0_399

Here are the summer and winter period vsdb verification:
https://www.emc.ncep.noaa.gov/gmb/jhan/vsdbw/progcw1/ (winter)
https://www.emc.ncep.noaa.gov/gmb/jhan/vsdbw/progcs1/ (summer)

### CODE REVIEW

**Review by ChunxiZhang-NOAA — APPROVED** [review]
Just some parameter tuning. Looks good.

**Comment by lisa-bengtsson** [comment]
@JongilHan66 please have a look at these three updated routines when you have a chance.

**Comment by lisa-bengtsson** [comment]
@ChunxiZhang-NOAA I updated the deep and shallow cu schemes following Jongil's suggestion after you approved the PR, you can have a look when you get a chance. Thank you.

**Comment by ChunxiZhang-NOAA** [comment]
@lisa-bengtsson Sure I will review it again. BTW:  It would be great if there are a few slides of PPT showing how the changes of the cumulus scheme impact the results. @JongilHan66 Do you have those slides?

**Comment by lisa-bengtsson** [comment]
> @lisa-bengtsson Sure I will review it again. BTW: It would be great if there are a few slides of PPT showing how the changes of the cumulus scheme impact the results. @JongilHan66 Do you have those slides?

@ChunxiZhang-NOAA I can update the slides I presented in the last UFS convection WG meeting with these last tests based on the tuning in this PR, do you think it is OK if that is done by Friday this week? This PR probably wont be merged before then?

**Comment by ChunxiZhang-NOAA** [comment]
@lisa-bengtsson Sure, please attach your results you presented last time. If @JongilHan66 could also attach his testing results for his changes, it would be great.  No problem, take your time. This PR is targeted to be merged around Nov 10.

**Comment by JongilHan66** [comment]
> 

@lisa-bengtsson The changes are correct

File: physics/progsigma_calc.f90
Comment (grantfirl) [inline physics/progsigma_calc.f90]: This is a coding style choice, but IMO, the new line here and at 227 can be combined with the previous lines since they're just putting a lower bound on the calculation of `sigmab`. Not a big deal though.

**Review by grantfirl — APPROVED** [review]

File: physics/samfshalcnv.f
Comment (mzhangw) [inline physics/samfshalcnv.f]: typo: extra "i"

Comment (lisa-bengtsson) [reply physics/samfshalcnv.f]: ah, thank you for catching that

**Comment by lisa-bengtsson** [comment]
@ChunxiZhang-NOAA is it possible to re-run the ccpp_build step for the CI test? I think it automatically ran in between updating the ccpp/physics submodule and the FV3 submodule. Thanks

**Comment by dustinswales** [comment]
> @ChunxiZhang-NOAA is it possible to re-run the ccpp_build step for the CI test? I think it automatically ran in between updating the ccpp/physics submodule and the FV3 submodule. Thanks

@lisa-bengtsson The CI tests have a bug in them, that's why they are failing. I [opened a tiny PR](https://github.com/lisa-bengtsson/ccpp-physics/pull/1/files) into this branch to address this. If you are okay with this we can include it in this PR? Also, @ChunxiZhang-NOAA @grantfirl Do you think it's okay we add this here? Or open a whole new PR?

**Comment by ChunxiZhang-NOAA** [comment]
@dustinswales I can rerun the ccpp_build step for the CI tests manually. But without your bug fix being merged in, the tests will fail again.

**Comment by dustinswales** [comment]
@ChunxiZhang-NOAA The SCM test will fail, since it's not up-to-date, but the fv3atm test will pass.

**Comment by grantfirl** [comment]
> NOAA-EMC/fv3atm#598

@dustinswales I think it's OK to add the bugfix here since we're discussing it here and it's just a minor bugfix.

**Comment by lisa-bengtsson** [comment]
It is Ok with me to add the bugfix here.

**Comment by ChunxiZhang-NOAA** [comment]
@dustinswales I will try fv3atm test then.  BTW: it is ok to add the minor bug fix here.

**Comment by dustinswales** [comment]
Sounds good.
@lisa-bengtsson You can go ahead and merge my PR into here.

**Comment by ChunxiZhang-NOAA** [comment]
@dustinswales The CI tests are successful after the bug fix was merged in.

**Comment by jkbk2004** [comment]
All tests are done on [ufs wm pr #1480](https://github.com/ufs-community/ufs-weather-model/pull/1480). Please, go ahead to merge in this pr.

**Pull request merged**
